test/testlib/minikube_utilities.go

```diff
@@ -733,7 +733,13 @@ func GetAppLog(t *testing.T, namespace string, podName string, fileNameSuffix st
 
 	writer := io.Writer(f)
 
-	reader := getAppLogStream(t, namespace, podName, podLogOptions)
+	reader, err := getAppLogStreamE(t, namespace, podName, podLogOptions)
```

Comment:
how did you hit an issue with this? We always ensure that a pod is running before we call GetAppLog on it.

Reply:
`TestAdminReadinessProbe` kills the `admin-0` Pod to verify that the `readinessprobe` script eventually fails on `admin-1`. We do not wait for `admin-0` to come back up, because we do not care about its state, we only want its presence in the Raft membership to prevent a leader from being elected.

Reply:
the issue here being that you terminate the test before admin-0 comes back up? That sounds unlikely.

Reply:
That's what happened. It only takes between 2-5 seconds for `admin-1` to become a CANDIDATE after `admin-0` is killed, and all of the remaining polling loops would complete in that case. From my experience, 2-5 seconds is not enough time for an Admin Pod to start up, but maybe that is only an issue when running on my machine.

Reply:
is this in the case when -1 is not the LEADER and hence you don't kill it?

Reply:
Yes, and that's the most likely scenario, since `admin-0` becomes initial leader and it is unlikely for a new leader election to be triggered while it's running.